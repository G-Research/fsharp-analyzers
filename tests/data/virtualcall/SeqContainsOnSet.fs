module M

let f () =
    let mySet = set [1..10]
    Seq.contains 2 mySet // should warn
