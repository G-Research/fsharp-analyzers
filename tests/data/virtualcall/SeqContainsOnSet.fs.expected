GRA-VIRTUALCALL-001 | Warning | (5,4 - 5,24) | Consider replacing the call of Seq.contains with a function from the Set module to avoid the costs of virtual calls.
